lib/assertions.js

```diff
@@ -735,9 +735,9 @@ module.exports = function (expect) {
     });
 
     expect.addAssertion([
-        '<array-like> to have items [exhaustively] satisfying <any+>',
+        '<array-like> to have items [exhaustively] satisfying <any>',
         '<array-like> to have items [exhaustively] satisfying <assertion>',
-        '<array-like> to be an array whose items [exhaustively] satisfy <any+>',
```

Comment:
In a good way :). Only suites that accidentally pass additional arguments (which are currently silently ignored) will break.